packages/peregrine/lib/talons/CheckoutPage/ShippingInformation/AddressForm/useCustomerForm.js

```diff
@@ -39,6 +47,30 @@ export const useCustomerForm = props => {
         }
     }, [getCustomerError]);
 
+    useEffect(() => {
+        if (createCustomerAddressError || updateCustomerAddressError) {
+            errorRef.current.scrollIntoView({
```

Comment:
I think `scrollIntoView()` is the right choice here, but I don't think it belongs in the talon.

The only other place we use this, [`PaymentsFormItems`](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/Checkout/paymentsFormItems.js#L55), has a talon of its own as well, but the component is the one to call `scrollIntoView` because this kind of effect is really just presentational. There's very little logic we put in the component rather than the talon, but I think this qualifies.

Reply:
Moved this logic to `FormError` and put scroll and ref logic in component instead of talon (57ab1a7).